lib/helium/collection.rb

```diff
@@ -80,6 +80,36 @@ def last
       collection.last
     end
 
+    # Adds resources of the same type to the collection related to the
+    # '@belongs_to' resource
+    def add_relationships(items)
+      body = relationship_request_body(items)
+
+      @client.post(relationship_path, body: body) if relationship_path
+
+      self
```

Comment:
I thought this was the case, because `each` and friends call `fetch_collection`, which fetches the most up-to-date representation of the collection. Am I wrong? Is something catched? My intention was to make it chainable.